Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -879,7 +879,7 @@ public function isShortArray($index)
 
         $prevToken = $this[$this->getPrevMeaningfulToken($index)];
 
-        if ($prevToken->equalsAny(array(array(T_DOUBLE_ARROW), '=', '+', '(', '['))) {
+        if ($prevToken->equalsAny(array(array(T_RETURN), array(T_DOUBLE_ARROW), '=', '+', '(', '['))) {
```

Comment:
It fixed one case, you found another.
I agree that this solution how we distinguish short vs long array is naive.
Do you know better approach ?